h2o-r/h2o-package/R/admissibleml.R

```diff
@@ -0,0 +1,150 @@
+#' Calculate intersectional fairness metrics.
+#'
+#' @param model H2O Model
+#' @param frame Frame used to calculate the metrics.
+#' @param protected_cols List of categorical columns that contain sensitive information
+#'                          such as race, gender, age etc.
+#' @param reference List of values corresponding to a reference for each protected columns.
+#'        If set to NULL, it will use the biggest group as the reference.
+#' @param favorable_class Positive/favorable outcome class of the response.
+#'
+#' @return Dictionary of frames. One frame is the overview, other frames contain dependence
+#'         of performance on threshold for each protected group.
+#' @export
```

Comment:
Can we add an example in the docstring?  I realize it will be a bit long... but maybe it would be helpful?